Docs/source/theory/picsar_theory.rst

```diff
@@ -1,36 +1,34 @@
-.. raw:: latex
```

Comment:
Can you rename this file to `pic_theory.rst`?
The picsar name is historic and not used anymore for the core routines.

Reply:
Sure